app/services/platform.rb

```diff
@@ -22,7 +22,9 @@ def self.connection
       f.request :retry, RETRY_OPTIONS
       f.adapter Faraday.default_adapter # this must be the last middleware
       f.ssl[:verify] = Rails.env.production?
-      f.basic_auth(*BASIC_AUTH) if BASIC_AUTH
+      if BASIC_AUTH && BASIC_AUTH[:username].present?
+        f.basic_auth(BASIC_AUTH[:username], BASIC_AUTH[:password])
+      end
```

Comment:
Mind adding a test?

Reply:
There wasn't one before, but there might be one now.